helm/values.yaml

```diff
@@ -1,33 +1,8 @@
-namespace: bitswap-peer
 image:
-  version: "20220513.1754"
   pullPolicy: Always
 service:
   port: 3000
-  metrics:
-    port: 3001
-    publicEndpoint: "true"
-    enableMonitor: "true"
-resources:
-  limits:
-    cpu: 4
-    memory: 8Gi
-  requests:
-    cpu: 3
-    memory: 2Gi
-autoscaling:
-  enabled: true
-  minReplicas: 2
-  maxReplicas: 40
-  targetCPUUtilizationPercentage: 70
-  targetMemoryUtilizationPercentage: 70
```

Comment:
Do we want to set sane defaults that envs could just override?

Reply:
Done!